docs/examples/JSONExample/src/JSONExample.ls

```diff
@@ -90,11 +123,51 @@ package
             sprite.x = 240;
             sprite.y = 120;
             stage.addChild(sprite);
+            
+            
+            var json = new JSON();
+            
+            json.initObject();
+            
+            
+            // Circular reference tests
```

Comment:
Ah, the commented out bit trips an assertion (equivalent to Flash throwing an error). I don't know a good way to handle that in an example, so I just left it commented and added a line explaining it.